backend/src/main/scala/cromwell/backend/BackendCacheHitCopyingActor.scala

```diff
@@ -5,21 +5,21 @@ import akka.event.LoggingReceive
 import cromwell.backend.BackendCacheHitCopyingActor.CopyOutputsCommand
 import cromwell.backend.BackendJobExecutionActor.{AbortedResponse, BackendJobExecutionResponse, FailedNonRetryableResponse}
 import cromwell.backend.BackendLifecycleActor._
-import cromwell.core.JobOutputs
+import cromwell.core.simpleton.WdlValueSimpleton
 
 import scala.concurrent.Future
 
 object BackendCacheHitCopyingActor {
-  final case class CopyOutputsCommand(outputs: JobOutputs)
+  final case class CopyOutputsCommand(wdlValueSimpletons: Seq[WdlValueSimpleton], jobDetritusFiles: Map[String,String], returnCode: Option[Int])
```

Comment:
If I were @geoffjentry I'd recommend a `Seq[JobDetritus]` ADT instead of this `Map` for detrituses